lib/getData.js

```diff
@@ -7,9 +7,9 @@ const getData = async configPath => {
     coverageThreshold: { global: thresholds },
     coverageDirectory = 'coverage',
   } = typeof config === 'function' ? await config() : config
+  const normalizedCoverageDirectory = coverageDirectory.replace('<rootDir>', path.dirname(configPath))
   const reportPath = path.resolve(
-    path.dirname(configPath),
-    coverageDirectory,
+    normalizedCoverageDirectory,
```

Comment:
Will this also work if `coverageDirectory` does _not_ contain `<rootDir>`?

Reply:
@rbardini, of course. In case path does not contain `<rootDir>`, the path will be used as is (no substitutions).

Reply:
I can add a test for that if you want

Reply:
When `coverageDirectory` does not contain `<rootDir>`, the change will be equivalent to:

```diff
 const reportPath = path.resolve(
-  path.dirname(configPath),
   coverageDirectory,
   'coverage-summary.json',
 )
```

I wonder if `resolvePath` will still resolve correctly without `configPath`.

What if we simply replaced `<rootDir>` with the current directory? Something like:

```diff
 const reportPath = path.resolve(
   path.dirname(configPath),
-  coverageDirectory,
+  coverageDirectory.replace('<rootDir>', '.'),
   'coverage-summary.json',
 )
```

Reply:
I see, yeah. You are right - let me work on tests as well.
Since the code tries to resolve the `coverage-summary.json` from the config path dir - this should work.

Reply:
@rbardini done